### ISSUE TITLE
Blind accessibility with screen readers in UI

### ISSUE THREAD

**Issue by RealAmethyst** [ryujinx-mirror/ryujinx#54]
Ryujinx UI is not scren reader friendly. This means that people who use screen readers to navigate the computer can't use it. A screen reader is a program that reads text out loud so that blind and low vision users can use a computer.
Is there a way to make screen readers work inside Ryujinx? There are games that blind people are able to play, but we are unable to configure the emulator, as we have no speech output.
